tests/client/ui/components/TestOverlay.js

```diff
@@ -39,6 +39,18 @@ describe('<Overlay />', () => {
     assert.include(root.overlayContainer.className, 'Overlay--visible');
   });
 
+  it('hides when you click the background', () => {
```

Comment:
Seems like an `Esc` `keyPress` would also be appropriate, especially for accessibility as I'm not sure there is a "close window" button.

That's my bad as I wrote the component, but I'm just realising it now.